src/ray/gcs/gcs_server/test/gcs_placement_group_scheduler_test.cc

```diff
@@ -50,19 +55,43 @@ class GcsPlacementGroupSchedulerTest : public ::testing::Test {
     thread_io_service_->join();
   }
 
-  template <typename Data>
-  void WaitPendingDone(const std::vector<Data> &data, int expected_count) {
-    auto condition = [this, &data, expected_count]() {
-      absl::MutexLock lock(&vector_mutex_);
-      return (int)data.size() == expected_count;
+  void WaitPlacementGroupPendingDone(unsigned int expected_count,
```

Comment:
```suggestion
  void WaitPlacementGroupPendingDone(int expected_count,
```

Reply:
the reason use  unsigned int is `GcsPlacementGroupStatus` enum is `uint32_t`,  I will change it to `int32_t`

Reply:
updated